### DISCUSSION TITLE
q-uploader capture set to environment not working

### DISCUSSION THREAD

**Discussion by warrence** [quasarframework/quasar#16495] [CLI - Capacitor / Cordova modes]
```vue
<q-uploader
    accept="image/*"
    capture="environment"
    :factory="upload"
    label="Upload Photos"
    multiple
    batch
    style="max-width: 100%; width: 100%"
    v-if="checkingHistoryId"
/>
```

Hello,

I'm currently working on a project where I'm trying to set the rear camera as the default option by using the "capture" attribute with the value "environment" in my Android app. However, I've noticed that, by default, it seems to use the front camera instead. Interestingly, I can manually switch to the rear camera within the camera screen, which suggests that the rear camera is functioning correctly.

Has anyone else come across this issue? this issue only happen in android, ios was working fine, and any insights or advice would be greatly appreciated. Thank you!

**Comment by pdanpdan** [comment]
try this https://codepen.io/pdanpdan/pen/qBgBExN on the mobile
if it works then there is a problem with QUploader
else there is one with browser/mobile

**Reply by warrence** [reply]
I tried, all the of them are opening the front camera.

**Reply by yusufkandemir** [reply]
The problem is unrelated to Quasar. `capture` itself works, but the values `user`/`environment` seems to work only for WebKit-based browsers. So, that's why it works on iOS but not on Android. See:
https://stackoverflow.com/a/57138976/9260314
https://stackoverflow.com/a/57189707/9260314
